src/main/java/seedu/duke/command/DeleteEventCommand.java

```diff
@@ -0,0 +1,47 @@
+package seedu.duke.command;
+
+import seedu.duke.data.exception.SystemException;
+import seedu.duke.data.timetable.Event;
+import seedu.duke.ui.InterfaceManager;
+
+import static seedu.duke.util.PrefixSyntax.PREFIX_DELIMITER;
+import static seedu.duke.util.PrefixSyntax.PREFIX_INDEX;
+
+/**
+ * This class allows us to store relevant information regarding the events to be deleted from the timetable and
+ * manipulate it before executing it via the execute method.
+ */
+public class DeleteEventCommand extends Command {
+
+    public static final String COMMAND_WORD = "delete-e";
+    public static final String COMMAND_SUCCESSFUL_MESSAGE = "Event deleted:" + InterfaceManager.LS;
+
+    public static final String COMMAND_USAGE = COMMAND_WORD + ": Deletes an event. Parameters: "
```

Comment:
This two should be private members.

Reply:
Alright. Apologies for the mistake. Was referring to an older version of the code base for styling and didn't change this part.